charts/formbricks/templates/cube-deployment.yaml

```
@@ -0,0 +1,108 @@
+      app.kubernetes.io/name: {{ include "formbricks.cubeName" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "formbricks.cubeName" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: cube
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/cube-configmap.yaml") . | sha256sum }}
+    spec:
+      {{- if .Values.deployment.imagePullSecrets }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)